f2/src/forms/HowDidItStartForm.js

```diff
@@ -22,13 +22,6 @@ export const HowDidItStartForm = props => {
   const [data] = useStateValue()
   const howdiditstart = {
     howDidTheyReachYou: [],
-    email: '',
-    phone: '',
-    online: '',
-    application: '',
-    others: '',
-    whenDidItStart: '',
-    howManyTimes: '',
     ...data.formData.howdiditstart,
```

Comment:
I think we need to keep these in - it's giving the app default values for the fields.

Reply:
Added